url-plugins/checkstyle.xml

```diff
@@ -0,0 +1,400 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
i was having issues building directly from the directory. What do i need to add to the pom.xml to reference the checkstyle.xml in the parent directory?